charts/rasa-x/templates/rasa-config-files-configmap.yaml

```diff
@@ -33,7 +33,7 @@ data:
       username: "{{ .Values.rabbitmq.rabbitmq.username }}"
       password: ${RABBITMQ_PASSWORD}
       port: {{ default 5672 .Values.rabbitmq.service.port }}
-      queue: ${RABBITMQ_QUEUE}
+      queues: ${RABBITMQ_QUEUE}
```

Comment:
@mprazz we should change it only if used version of Rasa OOS is >= version that is supporting `queues`.

(I've put `1.9.0` as an example :D )
```
{{- if semverCompare ">= 1.9.0" .Values.rasa.tag -}}
      queues: ${RABBITMQ_QUEUE}
{{- else -}}
      queue: ${RABBITMQ_QUEUE}
{{- end -}}
```

Reply:
Deprecation was introduced on `1.9.0` actually 😂 
Also added the list format for `queues`

Reply:
Cool, I fixed the template formating, the rest looks good 🚀